packages/core/src/interfaces/charts.ts

```diff
@@ -176,10 +181,10 @@ export interface LineChartOptions extends ScatterChartOptions {
 	 * options for the curve of the line
 	 */
 	curve?:
-		| string
-		| {
-				name: string;
-		  };
+	| string
+	| {
+		name: string;
+	};
```

Comment:
Can we make this one line? Same comment as before about the `{ useBBox: true }`

Reply:
prettier puts it like thissssss